model/src/main/kotlin/utils/OrtResultExtensions.kt

```diff
@@ -88,5 +88,5 @@ fun OrtResult.getDetectedLicensesWithCopyrights(
 /**
  * Copy this [OrtResult] and add all [labels] to the existing labels, overwriting existing labels on conflict.
  */
-fun OrtResult.mergeLabels(labels: Map<String, String>) =
-    copy(labels = this.labels + labels).apply { data += this@mergeLabels.data }
+fun OrtResult.mergeLabels(labels: List<String>) =
```

Comment:
This should be a `Collection`, `Set`, or `SortedSet` now, but not a `List`.

Reply:
Ideally `Collection` imo.

Reply:
Done.